docker-compose.yaml

```diff
@@ -16,7 +16,6 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-version: '3.0'
```

Comment:
Why do we remove this?

Reply:
Because there's warning when launch playground: WARN[0000] .../docker-compose.yaml: `version` is obsolete

Reply:
OK.